chain/client/src/test_utils.rs

```diff
@@ -903,9 +903,14 @@ pub fn setup_mock_all_validators(
                             if aa.get(&key).is_none() {
                                 aa.insert(key);
                                 for (_, view_client) in connectors1.read().unwrap().iter() {
-                                    view_client.do_send(NetworkViewClientMessages::AnnounceAccount(
-                                        vec![(announce_account.clone(), None)],
-                                    ))
+                                    let view_client = view_client.clone();
+                                    let announce_account = announce_account.clone();
```

Comment:
I'd move the clone() calls as close to the definition of the async block as possible.
With a bit of mind gymnastics, this way you may convince yourself that then it looks as if you were listing arguments of an async lambda function (which I think is neat, something like [...](){} syntax in C++) ).
In this case it would be:

ctx.spawn(wrap_future({
  let view_client = ...clone();
  let announce_account = ...clone();
  async move {...}
}));